src/frontend/src/components/public/PIAFormTabs/intake/index.tsx

```diff
@@ -37,18 +44,90 @@ export const PIAFormIntake = () => {
 
   return (
     <>
-      <section className="">
+      <section>
         <h2>{Messages.PiaIntakeHeader.H1Text.en} </h2>
-        <h3>{Messages.PiaIntakeHeader.H2Text.en} </h3>
-        <ul>
-          {Messages.PiaIntakeHeader.ListText.map((item, index) => (
-            <li key={index}>{item.en}</li>
-          ))}
-        </ul>
+        {!isReadOnly && (
+          <>
+            <h3>{Messages.PiaIntakeHeader.H2Text.en} </h3>
+            <ul>
+              {Messages.PiaIntakeHeader.ListText.map((item, index) => (
+                <li key={index}>{item.en}</li>
+              ))}
+            </ul>
+          </>
+        )}
       </section>
-      <form className="needs-validation">
-        <section className="section__padding-block">
-          <h3>{Messages.GeneralInfoSection.H2Text.en}</h3>
+
+      <section className="section__padding-block">
+        <h3>{Messages.GeneralInfoSection.H2Text.en}</h3>
+        {isReadOnly ? (
+          <div className="card p-3">
+            <div className="container d-grid gap-3">
+              <div>
+                <div className="row ">
+                  <div className="col col-md-3">
+                    <b>Drafter</b>
+                  </div>
+                  <div className="col col-md-3">
+                    <b>Ministry</b>
+                  </div>
+                  <div className="col col-md-3">
+                    <b>Branch</b>
+                  </div>
+                  <div className="col col-md-3">
+                    <b>Initiative Lead</b>
+                  </div>
+                </div>
+                <div className="row">
+                  <div className="col col-md-3">{pia.drafterName}</div>
+                  <div className="col col-md-3">{piaMinistryFullName}</div>
+                  <div className="col col-md-3">{pia.branch}</div>
+                  <div className="col col-md-3">{pia.mpoName}</div>
+                </div>
+                <div className="row">
+                  <div className="col col-md-3">{pia.drafterTitle}</div>
+                  <div className="col col-md-3"></div>
+                  <div className="col col-md-3"></div>
+                  <div className="col col-md-3">{pia.leadTitle}</div>
+                </div>
+                <div className="row">
+                  <div className="col col-md-3">{pia.drafterEmail}</div>
+                  <div className="col col-md-3"></div>
+                  <div className="col col-md-3"></div>
+                  <div className="col col-md-3">{pia.leadEmail}</div>
```

Comment:
why scattered data? You can have row inside a column as well - making it nested.